docs/src/content/docs/en/installation.md

```diff
@@ -35,10 +35,18 @@ After your app has been initialized, be sure to follow any instructions outputed
 
 The following would scaffold a CES app with Expo Router, NativeWind, and Bun:
 
+### npm
+
 ```bash
 npm create expo-stack --expo-router --nativewind --bun
 ```
 
+### bun
+
+```bash
+bun create expo-stack <destination> --expo-router --nativewind --bun
```

Comment:
I'm a bit confused by the `<destination>` bit, can you elaborate?

Reply:
My bad. I've learned a bit more about expo-stack and realised some stuff in the meanwhile.
`bun create` has two arguments: the template (`expo-stack`) and the destination (of where the template is cloned). The cli uses the second argument for the project name.
So for example if I am in root and want to create the app in Documents folder I write the following:
`bun create expo-stack ./Documents/test-app`

But the cli takes the destination and uses it as the project name, thus app.json name, slug and scheme will be `./Documents/test-app`

Reply:
Now, I've updated the cli to split the first param by `/` and always take the last segment of the path

Reply:
While keeping the logic for splitting the first param by `/` is a good idea, I think the documentation should suggest that "destination" argument is the app name so it should be:
`bun create expo-stack myapp --expo-router --nativewind --bun`